stewart/theme.json

```diff
@@ -0,0 +1,361 @@
+{
+	"version": 1,
+	"templateParts": [
+		{
+			"name": "header",
+			"title": "Header",
+			"area": "header"
+		},
+		{
+			"name": "header-spacer",
+			"title": "Header Spacer",
+			"area": "header"
+		},
+		{
+			"name": "sidebar",
+			"title": "Sidebar",
+			"area": "sidebar"
+		},
+		{
+			"name": "footer",
+			"title": "Footer",
+			"area": "footer"
+		}
+	],
+	"customTemplates": [
+		{
+			"name": "blank",
+			"title": "Blank",
+			"postTypes": [
+				"page",
+				"post"
+			]
+		},
+		{
+			"name": "header-footer-only",
+			"title": "Header and Footer Only",
+			"postTypes": [
+				"page",
+				"post"
+			]
+		},
+		{
+			"name": "sidebar-footer-only",
+			"title": "Sidebar and Footer Only",
+			"postTypes": [
+				"page",
+				"post"
+			]
+		}
+	],
+	"settings": {
+		"appearanceTools": true,
+		"color": {
+			"duotone": [
+				{
+					"colors": [ "#F1F2F2", "#F1F2F2" ],
+					"slug": "background-and-foreground",
+					"name": "Background and Foreground"
+				},
+				{
+					"colors": [ "#C14420", "#F1F2F2" ],
+					"slug": "foreground-and-background",
+					"name": "Foreground and background"
+				}
+			],
+			"palette": [
```

Comment:
Looks like the palette colors `primary` `secondary` and `selection` aren't used at all, perhaps they should be removed to avoid confusion.

<img src="https://user-images.githubusercontent.com/146530/145284305-4870a4af-0840-410c-b387-af2d1a281873.png" width=200>

Reply:
Primary is actually used in a couple places (like hovers). In any case, I just went in and rearranged these so they the most often used colors are up front. I also replaced "selection" with "tertiary" so that these colors match up with the ones we're using in our Blockbase themes. That way if the user has existing patterns that use these color names, they'll translate over just fine.